netplugin/netd.go

```diff
@@ -322,6 +343,11 @@ func processStateEvent(netPlugin *plugin.NetPlugin, opts cliOpts, rsps chan core
 				log.Infof("Received %q for Bgp: %q", eventStr, bgpCfg.Hostname)
 				processBgpEvent(netPlugin, opts, bgpCfg.Hostname, isDelete)
 				continue
+			}
+			if epgCfg, ok := currentState.(*mastercfg.EndpointGroupState); ok {
```

Comment:
This should be handled in processCurrentState as well. Which will be need in netplugin restart cases.